src/app/stylesheets/custom/button.less

```diff
@@ -42,15 +42,51 @@
     }
   }
 
+  &.btn-info {
+    color: @btn-text-color;
+    background-color: @btn-bg-color;
+    border-color: @btn-border-color;
+  }
+
+  &.btn-danger {
+    color: @btn-danger-text-color;
```

Comment:
For the button color, bootstrap already use color variables: http://getbootstrap.com/css/#less-variables-colors

It would be a good idea to keep the button semantic (default, info, warning, danger, ...) and override those default colours if needed.